src/py/pyodide/webloop.py

```diff
@@ -143,7 +143,13 @@ def call_later(
         if delay < 0:
             raise ValueError("Can't schedule in the past")
         h = asyncio.Handle(callback, args, self, context=context)
-        setTimeout(create_once_callable(h._run), delay * 1000)
+
+        def run_handle():
+            if h._cancelled:
```

Comment:
Isn't the public metod `cancelled()` instead? https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Handle.cancelled

Reply:
I guess so. I was copying from `base_events.py` which uses `_cancelled`, but this seems better.